lib/jwt_signed_request.rb

```diff
@@ -28,16 +32,20 @@ def self.sign(method:, path:, body: EMPTY_BODY, headers:, secret_key:, algorithm
     )
   end
 
-  def self.verify(request:, secret_key:, algorithm: nil)
+  def self.verify(request:, secret_key:, algorithm: nil, leeway: 0)
```

Comment:
for the default leeway you mean? Good q. In fact looking at it again I think we should not pass the option when the leeway param is not provided. JWT has its own default (which happens to be 0).